client/coral-embed-stream/src/containers/Embed.js

```diff
@@ -94,25 +92,24 @@ const mapStateToProps = state => ({
   commentId: state.stream.commentId,
   assetId: state.stream.assetId,
   assetUrl: state.stream.assetUrl,
-  activeTab: state.embed.activeTab,
+  activeTab: state.embed.activeTab
 });
 
 const mapDispatchToProps = dispatch =>
-  bindActionCreators({
-    fetchAssetSuccess,
-    checkLogin,
-    setCommentCountCache,
-    viewAllComments,
-    logout,
-    setActiveTab,
-  }, dispatch);
+  bindActionCreators(
+    {
+      logout,
+      checkLogin,
+      setActiveTab,
+      viewAllComments,
+      fetchAssetSuccess,
+      setCommentCountCache
+    },
+    dispatch
+  );
 
 export default compose(
   connect(mapStateToProps, mapDispatchToProps),
-  branch(
-    props => !props.auth.checkedInitialLogin,
-    renderComponent(Spinner),
-  ),
-  withQuery,
+  branch(props => !props.auth.checkedInitialLogin && props.config, renderComponent(Spinner)),
```

Comment:
Where does `props.config` come from? I think you forgot to add it in `mapStateToProps`.